src/module/actor/sheet/base.ts

```diff
@@ -462,6 +468,13 @@ abstract class ActorSheetPF2e<TActor extends ActorPF2e> extends ActorSheet<TActo
         }
     }
 
+    protected _resetListeners(): void {
```

Comment:
Since we're using typescript, underscore-leading method and property names generally aren't needed.